### PR TITLE
Added README and Code of Conduct files

### LINKED ISSUE #8: Add readme and code of conduct.

### PR DESCRIPTION
**Pull request by marianoaguero**
## Description

Added README and Code of Conduct files

## Story/Task/Issue ID
Copy the code number for the planned item that is implemented by this PR.

## How Has This Been Tested?

Open README.md and code_of_conduct.md files.

**Test Configuration**:
* Device Android version:
* Phone Maker:
* Phone Model:

## Checklist:
 - [ ] Code compiles and runs on my devices.
 - [ ] Changes meet acceptance criteria.
 - [ ] Self-review of my own code.
 - [ ] Code is commented where needed.
 - [ ] Conventional commits with meaningful messages.
 - [x] Changes to the docs.
 - [ ] My changes generate no new warnings.
 - [ ] Added/Changed tests.
 - [ ] All tests pass after changes.
 - [ ] Squash to minimum of commits.
 - [x] PR title, description and labels for humans.
 - [x] Reviewer/s assigned.

## Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (estebandres) [inline code_of_conduct.md]: contact email maybe?

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+## Branching
+This projects uses the [Git Flow](https://guides.github.com/introduction/flow/) workflow with a Fork & Pull Model
+
+## Contributing
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
+
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
```
Comment (estebandres) [inline README.md]: Please make sure to add and update tests related to your changes.

**Review by estebandres — CHANGES_REQUESTED** [review]
small things

**Review by estebandres — APPROVED** [review]
all good

**Review by miglierinicarlos — APPROVED** [review]
No veo errores

**Pull request merged**